checker/tests/lock/Issue524.java

```diff
@@ -0,0 +1,18 @@
+import org.checkerframework.checker.lock.qual.*;
+import org.checkerframework.dataflow.qual.*;
+
+//https://github.com/typetools/checker-framework/issues/524
```

Comment:
Most other issue test cases use this "header style":

// Test case for Issue 471:
// https://github.com/typetools/checker-framework/issues/471